### ISSUE TITLE
[BUG] redis: "Connection is closed"

### ISSUE THREAD

**Issue by Spongman** [cweijan/vscode-mysql#587]
connected to Ubuntu wsl1 instance via remote-wsl.

from wsl, i can connect to the redis instance:
```
$ nc -vz localhost 16379
Connection to localhost (127.0.0.1) 16379 port [tcp/*] succeeded!
```

the "Open Terminal" link in the nosql UI works ok:

![image](https://user-images.githubusercontent.com/1088194/169166186-24a4f521-363d-4bdb-8419-d4dbd8270f7b.png)

however, the nosql UI itself just shows "Connection is closed":

![image](https://user-images.githubusercontent.com/1088194/169166242-093cbe76-9155-4b03-93a5-c24812c65452.png)

```
[2022-05-18 15:25:06.916] [renderer14] [error] [Extension Host] Trace: [Error: Connection is closed.
	at EventEmitter.sendCommand (/home/user/.vscode-server/extensions/cweijan.vscode-database-client2-5.3.0/out/extension.js:2:2563339)
	at EventEmitter.info (/home/user/.vscode-server/extensions/cweijan.vscode-database-client2-5.3.0/out/extension.js:2:2618175)
	at S.<anonymous> (/home/user/.vscode-server/extensions/cweijan.vscode-database-client2-5.3.0/out/extension.js:2:5210821)
	at Generator.next (<anonymous>)
	at s (/home/user/.vscode-server/extensions/cweijan.vscode-database-client2-5.3.0/out/extension.js:2:5209513)
	at runMicrotasks (<anonymous>)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)]
	at Function.log (/home/user/.vscode-server/extensions/cweijan.vscode-database-client2-5.3.0/out/extension.js:2:4911803)
	at /home/user/.vscode-server/extensions/cweijan.vscode-database-client2-5.3.0/out/extension.js:2:4929984
	at runMicrotasks (<anonymous>)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)```

shouldn't that be run in the 'Remote Extension Host' ? seeing that error in the renderer seems wrong.

- OS: win10x64, Ubuntu
- Database Type: redis

**Comment by cweijan** [comment]
Hi, does "Connection is closed" happen after opening the terminal?

**Comment by Spongman** [comment]
yes, it happens both before & after opening the terminal.

**Comment by cweijan** [comment]
I have fixed this in version 5.3.1.
